scripts/create-iso.sh

```diff
@@ -236,6 +236,11 @@ if [ $DOREPO = 1 ]; then
     createrepo_c ${VERBOSE} "$ISODIR"
     if [ -n "$SIGNSCRIPT" ]; then
         "$SIGNSCRIPT" "$ISODIR"
+        # Check that the digest is strong enough. Value 8 means SHA256.
+        # See https://www.rfc-editor.org/rfc/rfc4880#section-9.4
+        echo "Checking the strength of the signature (repodata/repomd.xml.asc)"
+        gpg --list-packets "$ISODIR"/repodata/repomd.xml.asc |
+            awk '/digest algo/ { if ($3 >= 8) { print "Valid digest algorithm"; exit 0; } else { print "Invalid digest algorithm"; exit 1; } }'
```

Comment:
this would allow not-yet assigned algo IDs (which *may* be OK), but also includes the private/experimental range (which I think is not)

Reply:
To what range would you want to restrict? 8-11?

Reply:
if we want to be extra safe, I'd say yes